ql.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2014-2015 Jakob Borg and Contributors (see the CONTRIBUTORS file).
```

Comment:
"(c) Audrius" or something more general on all of these, I have no hand in this I think. :)